README.md

```diff
@@ -59,11 +59,13 @@ see the [SDK section of the book](https://anton-rs.github.io/kona/sdk/intro.html
 
 - [`common`](./crates/common): A suite of utilities for developing `client` programs to be run on top of Fault Proof VMs.
 - [`common-proc`](./crates/common-proc): Proc macro for the `client` program entrypoint.
-- [`primitives`](./crates/primitives): Primitive types for use in `kona` crates.
 - [`preimage`](./crates/preimage): High level interfaces to the [`PreimageOracle`][fpp-specs] ABI
+
+**Protocol**
```

Comment:
Pipeline

Reply:
Also has the implementation of the executor + Ethereum MPT. Not just pipeline